ci/build_container/build_recipes/versions.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+
+# Please ensure edits to this file don't break the python script 'get_repository_info.py'
+# ie. from root of the project run
+#
+# python bazel/print_dependencies.py
+#
+# and the bazel run rule
+#
+# bazel run //bazel:print_dependencies
+#
+# Since the script use's python's os.path.expandvars we do not quote variables
+# in this file.
+
+# use commit where cmake 3.6 feature removed. Unblocks Ubuntu 16.xx or below builds
+# TODO (moderation) change back to tarball method on next benchmark release
+BENCHMARK_GIT_SHA=505be96ab23056580a3a2315abba048f4428b04e
+BENCHMARK_FILE_URL=https://github.com/google/benchmark/archive/$BENCHMARK_GIT_SHA.tar.gz
+BENCHMARK_FILE_SHA256=0de43b6eaddd356f1d6cd164f73f37faf2f6c96fd684e1f7ea543ce49c1d144e
+BENCHMARK_FILE_PREFIX=benchmark-$BENCHMARK_GIT_SHA
+
+CARES_VERSION=1.15.0
+CARES_TAG=cares-1_15_0
+CARES_FILE_URL=https://github.com/c-ares/c-ares/releases/download/$CARES_TAG/c-ares-$CARES_VERSION.tar.gz
+CARES_FILE_SHA256=6cdb97871f2930530c97deb7cf5c8fa4be5a0b02c7cea6e7c7667672a39d6852
+CARES_FILE_PREFIX=c-ares-$CARES_VERSION
+
+GPERFTOOLS_VERSION=2.7
+GPERFTOOLS_TAG=gperftools-$GPERFTOOLS_VERSION
+GPERFTOOLS_FILE_URL=https://github.com/gperftools/gperftools/releases/download/$GPERFTOOLS_TAG/$GPERFTOOLS_TAG.tar.gz
```

Comment:
Nit: Google style requires `X="$Y"` when you are expanding out variables, so best to just do this everywhere in this file.